lib/straight/order.rb

```diff
@@ -163,6 +163,7 @@ def status_changed?
         @status_changed
       end
 
+      # @deprecated
       def tid
```

Comment:
Ah, ok. In fact, we can remove this method, while keeping this field in straight-server.